README.md

```diff
@@ -1,25 +1,48 @@
 # Setup
 
-Install Bitbar:
+## Fork and clone this repo
+
+Fork [`lewagon/bitbar`](https://github.com/lewagon/bitbar) to your own GitHub account and clone it on your computer:
```

Comment:
I think it's better to make people fork the repo so everyone can version it's own plugins.

Reply:
Good idea! I've turned this repo into a template, so no need to fork, people can use that button:

<img width="158" alt="image" src="https://user-images.githubusercontent.com/414418/103905357-ca3f6680-50fe-11eb-964c-96d55375e95e.png">

Reply:
The problem with the `Use this template` button is that users can choose whatever name they want for their repo. 

![image](https://user-images.githubusercontent.com/37899667/104005744-e21df580-51a5-11eb-9caf-4e3fbcc3bda6.png)

So the command below might not work if they choose a different name than `bitbar`. 

Plus the `upstream` remote will not be configured so they can't sync their own repo with `lewagon/bitbar` in case of change.

Reply:
I propose to keep the fork instructions to make the installation process more robust.